### PR TITLE
Gatsby v2

### PR DESCRIPTION
**Pull request by jquense**
should fix some issues where yarn toook forever to install deps.

### CODE REVIEW

File: .babelrc.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    [
+      '@babel/preset-react',
+      { development: process.env.NODE_ENV !== 'production' }
+    ]
+  ],
+  plugins: [
+    ['@babel/plugin-proposal-class-properties', { loose: true }],
+    '@babel/plugin-proposal-export-default-from',
+    '@babel/plugin-proposal-export-namespace-from',
+    '@babel/plugin-transform-runtime',
```
Comment (taion) [inline .babelrc.js]: `useESModules`?

File: www/gatsby-config.js [hunk 2]
```diff
@@ -5,7 +5,13 @@ const { addBootstrapPropTypes } = require('./bsPropUtils');
 module.exports = {
   siteMetadata: {
     title: 'React-Bootstrap Documentation',
-    author: 'Jason Quense'
+    author: 'Jason Quense',
+    browsers: [
```
Comment (taion) [inline www/gatsby-config.js]: why different from library browser support?

File: www/package.json [hunk 3]
```diff
@@ -9,22 +9,20 @@
...
-    "gatsby-plugin-catch-links": "^1.0.13",
-    "gatsby-plugin-less": "^1.0.7",
-    "gatsby-plugin-offline": "^1.0.11",
-    "gatsby-remark-prismjs": "^1.2.10",
-    "gatsby-source-filesystem": "^1.5.9",
-    "gatsby-transformer-react-docgen": "^1.0.8",
-    "gatsby-transformer-remark": "^1.7.22",
+    "gatsby-plugin-catch-links": "2.0.1-1",
+    "gatsby-plugin-less": "2.0.0-alpha.3",
+    "gatsby-remark-prismjs": "1.2.17",
```
Comment (taion) [inline www/package.json]: caret range?

File: www/gatsby-config.js (hunk 2 above)
Comment (jquense) [reply www/gatsby-config.js]: Idk

File: .travis.yml [hunk 4]
```diff
@@ -20,7 +20,7 @@ before_install:
   - sh -e /etc/init.d/xvfb start
 
 install:
-  - yarn && yarn run bootstrap-docs
+  - yarn run bootstrap
```
Comment (taion) [inline .travis.yml]: you can just do `yarn bootstrap`

**Pull request merged**